### ISSUE TITLE
VCL compiling error on parenthesis

### ISSUE THREAD

**Issue by rezan** [varnishcache/varnish-cache#2727]
Varnish cannot compile VCL which uses parenthesis in an expression. Here is a VTC:

```vcl
varnishtest "Test use of parens"

server s1 {
	rxreq
	txresp
} -start

varnish v1 -vcl+backend {
	sub vcl_recv {
		if ((2 - 1) > 0) {
			# do nothing
		}
	}
} -start

client c1 {
	txreq
	rxresp
	expect resp.status == 200
} -run
```

Here is the VCL compiling output:

```
#  Expected ')' got '>'
#  (program line 68), at
#  ('<vcl.inline>' Line 6 Pos 29)
#                  if ((2 - 1) > 0) {
#  ----------------------------#-----
#
#  Running VCC-compiler failed, exited with 2
#  VCL compilation failed
```

## Expected Behavior

Using 4.1, this works.

## Current Behavior

Fails on master.

**Comment by Dridi** [comment]
I reported this on IRC a while ago and @bsdphk wasn't interested in GCC's and Clang's arms race to more (and sometimes silly) warnings. You can get the same result without explicit parenthesis like in your example, which was the problem I ran into at the time.

**Comment by Dridi** [comment]
Oh, my bad, this is actually libvcc complaining here.

**Comment by bsdphk** [comment]
Fixed in 47dc5fd732eddff135d7c75672f2d3e28f324705
